engine/language_client_typescript/cli.js

```diff
@@ -5,6 +5,6 @@ if (require.main === module) {
     process.env.BAML_LOG = 'info'
   }
 
-  const baml = require('./native')
+  const baml = require('./artifacts/native')
```

Comment:
Can we split this change out of this PR?

I tried to do something along these lines a while back and screwed up the TS release in a hard-to-understand way. We can try another PR to push this forward but I'm uncomfortable doing that in this PR.

Reply:
Okay, sounds good. I did it in order to do some testing locally where I need my integ-test/react project to reference the node wasm bindings and this is how I got it to work.